src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colors getRandomColor() {
+        int index = new Random().nextInt((Colors.values().length));
```

Comment:
Make Random a class-level variable.

Reply:
Make Random a class-level variable. - I didnt understand... Could you explaine more about it.

Reply:
I didn't understand what I should to change, could you explaine more pls